web/resources/lib/components/dashboard/Dashboard.svelte

```diff
@@ -0,0 +1,115 @@
+<script lang="ts" context="module">
+  import { route } from "@wikijump/api"
+
+  /**
+   * Returns a dashboard route.
+   *
+   * @param path - The subpath.
+   */
+  export function dashboardRoute(path = "") {
+    if (path.startsWith("/")) path = path.substring(1)
+    return route("dashboard", { path })
+  }
+</script>
+
+<script lang="ts">
+  import Locale from "@wikijump/fluent"
+  import { Route } from "tinro"
+  import RouteAnnouncer from "../RouteAnnouncer.svelte"
+  import DashboardLink from "./DashboardLink.svelte"
+  import DashboardPanel from "./DashboardPanel.svelte"
+  import ProfilePanel from "./panels/ProfilePanel.svelte"
+  import SettingsPanel from "./panels/SettingsPanel.svelte"
+
+  const t = Locale.makeComponentFormatter("dashboard")
+
+  // bit of a hack, unfortunately, but this is to prevent Tinro
```

Comment:
I'm assuming here Tinro is the thing doing routing for the settings SPA?

Reply:
Yes, it's a very tiny router.